daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionCoderSpec.scala

```diff
@@ -39,7 +39,7 @@ class TransactionCoderSpec
 
   "encode-decode" should {
 
-    "do contractInstance" in {
+    "do contractInstance" ignore {
```

Comment:
Why is encoding/decoding flaky? That sounds very suspicious

Reply:
reverted